### PR TITLE
Fixes state cert tests

### PR DESCRIPTION
**Pull request by elena-kolevska**
- [X] Memcahced
- [x] Cosmos DB
- [x] SQL Server

### CODE REVIEW

File: tests/certification/state/memcached/memcached_test.go [hunk 1]
```diff
@@ -168,7 +169,7 @@ func TestMemcached(t *testing.T) {
 		key := certificationTestPrefix + "_expiresInOneSecondKey"
 		value := "This key will self-destroy in 1 second"
 
-		ttlExpirationTime := 1 * time.Second
+		ttlExpirationTime := 3 * time.Second
```
Comment (elena-kolevska) [inline tests/certification/state/memcached/memcached_test.go:172]: Expiration time is too short, the test was failing sporadically.

**Review by JoshVanL — APPROVED** [review]

**Review by antontroshin — APPROVED** [review]

**Pull request merged**